serving/src/main/java/feast/serving/config/ServingServiceConfig.java

```diff
@@ -82,6 +87,32 @@ public ServingService servingService(
     return servingService;
   }
 
+  @Bean
+  public HistoricalRetriever createJdbcHistoricalRetriever(FeastProperties feastProperties) {
+    FeastProperties.Store store = feastProperties.getActiveStore();
+    Map<String, String> config = store.getConfig();
+    String className = config.get("class_name");
```

Comment:
I'm not sure whether this is the best way to get the **class_name**?

Reply:
should be fine. use camel casing for the key.

Reply:
Are you saying we should do `String className = config.get("className");`?

Reply:
Oh understood, somebody else created the key and config. nvm Let it be for now.

Reply:
Yeah, my questions is that I'm not sure how do we set the config in the store `Map<String, String> config = store.getConfig();`? What is the exact key name of the `className`, "className" or "class_name"?